src/acftbias/generate_aircraft_bias_csv.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python3
+
+import csv
+
+# csv_write: fill a CSV file with coordinate-variable and draw-variable data for JEDI DrawValueFromFile
+#
+# INPUTS
+#
+# csvFileName: name of a write-allowable file for CSV output, will be generated or clobbered (string)
+# coefFileName: name of a read-allowable space-delimited GDAS bias correction file (string)
+# coordVariableName: <Group>/<name> of coordinate variable (string)
+# coordVariableType: variable-type for coordinate variable (string)
+# coordVariableColumn: column-number for coordinate variable in coefFile (integer)
+# drawVariableName: <Group>/<name> of draw variable (string)
+# drawVariableType: variable-type for draw variable (string)
+# drawVariableColumn: column-number for draw variable in coefFile (integer)
+#
+# OUTPUTS
+#
+# no returned variables, but CSV file is properly written and formatted for JEDI DrawValueFromFile
+#
+# DEPENDENCIES
+#
+# csv
+#
+def csv_write(csvFileName, coefFileName, coordVariableName, coordVariableType, coordVariableColumn,
+              drawVariableName, drawVariableType, drawVariableColumn):
+    import csv
```

Comment:
I think the second `import` is redundant here

Reply:
fixed